.github/workflows/pull-request.yml

```
@@ -194,23 +194,23 @@
         id: changed-files
         uses: tj-actions/changed-files@v40
-          
+
       - name: Run tests
         if: steps.changed-files.outputs.only_changed != 'true'
-        run: npm run test -- --watchAll=false --coverage       
-      
+        run: npm run test -- --watchAll=false --coverage
+
       - name: TypeScript compilation for changed files
         run: |
```

actionlint

shellcheck reported issue in this script: SC2043:warning:1:13: This loop will only ever run once. Bad quoting or missing glob/expansion?

(shellcheck)